src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -414,6 +428,94 @@ public String call() throws Exception {
         return response;
     }
 
+    private String putRequest(String path, String status) throws UnsupportedEncodingException {
+        logger.log(Level.FINEST, "PR-PUT-REQUEST:" + path + " with: " + status);
+        HttpClient client = getHttpClient();
+	client.getState().setCredentials(AuthScope.ANY, credentials);
+
+        PutMethod httpput = new PutMethod(path);
+        //http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html; section 14.10.
+        //tells the server that we want it to close the connection when it has sent the response.
+        //address large amount of close_wait sockets client and fin sockets server side
```

Comment:
Please use complete sentences in large comments and add space after `//` for readability.

Reply:
The same comment is used also in 2 other places. Do you think it should be also fixed there? Btw it just a extract from html rfc about closing connection - IMHO this comment can be removed entirely.

Reply:
Let's remove it then.